packages/extension/README.md

```diff
@@ -6,12 +6,20 @@ This is the WebExtension responsible for automating the browser
 
 ### Watching
 
+Kicks off the gulp watcher that rebuilds the app/lib directories on change.
+
 ```bash
 yarn workspace @packages/extension watch
 ```
 
 ## Building
 
+`@packages/extension` has a few different build processes occurring that are all driven by the [`gulpfile`](./gulpfile.ts).
+* `app` - The web extension piece of the code, has two separate bundles:
+    * `v2`: Version 2 of the web extension which uses [webpack](./webpack.config.mjs) to bundle the `app/v2` directory and output it as `background.js`
```

Comment:
Do we need v2 still? Wasn't this at the time kept to support old versions of Chrome that we no longer support?

Reply:
K, I was hoping this wasn't the case still :(

Reply:
I need to review bugzilla when I get a chance. @whimboo recently reached out about some of these issue. Right now we are really only use like 2-3 things for the v2 extension that we could hopefully soon move to BiDi and sunset it